README.md

```diff
@@ -1,7 +1,16 @@
 # Aker - Study management GUI
 
+# Installation
+## Dev environment
+1. Configure or update ports to services in `development.rb`.
+2. Setup DB using `rake db:setup`. Alternatively, use:
```

Comment:
Rake -> rails

Reply:
Yeah... The README needs a little refactor.. :P